pkg/joinserver/grpc_application_activation_settings_registry.go

```diff
@@ -76,22 +82,8 @@ func (srv applicationActivationSettingsRegistryServer) Set(ctx context.Context,
 		)
 	}
 	v, err := srv.JS.applicationActivationSettings.SetByID(ctx, req.ApplicationIdentifiers, req.FieldMask.Paths, func(stored *ttnpb.ApplicationActivationSettings) (*ttnpb.ApplicationActivationSettings, []string, error) {
-		if stored == nil {
-			if err := ttnpb.RequireFields(req.FieldMask.Paths,
-				"kek_label",
```

Comment:
`KEKLabel` was marked as required field in original issue, did that requirement change?

Reply:
It's not marked required, but it's also not marked explicitly optional in [the original issue](https://github.com/TheThingsIndustries/lorawan-stack/issues/2322). However, it does say "[...] multiple KEKs per application; at most one is fine", implying that it's optional. Anyway, it's optional now: users can also specify the default NetID only.